docs/getting-started/quick-start.md

```diff
@@ -184,9 +184,35 @@ fun RootContent(component: RootComponent, modifier: Modifier = Modifier) {
 }
 ```
 
+### Child Stack with SwiftUI
+
+```swift
+struct RootView: View {
+    private let root: RootComponent
+    
+    @ObservedObject
+    private var childStack: ObservableValue<ChildStack<AnyObject, RootComponentChild>>
+    
+    private var activeChild: RootComponentChild { childStack.value.active.instance }
+    
+    init(_ root: RootComponent) {
+        self.root = root
+        childStack = ObservableValue(root.childStack)
+    }
+    
+    var body: some View {
+        switch activeChild {
+        case let child as RootComponentChild.ListChild: ListView(child.component)
```

Comment:
Maybe reference and use [StackView](https://github.com/arkivanov/Decompose/blob/master/sample/app-ios/app-ios/StackView.swift) from samples?

Reply:
I think that using StackView will make it more complex and it requires a lot of code ?

Reply:
Sorry, I meant not to put the entire `StackView` code in the docs, but just use `StackView` and reference its source code from the samples. The same way as we use `ObservableValue` here. Using `StackView` [looks pretty simple](https://github.com/arkivanov/Decompose/blob/master/sample/app-ios/app-ios/CountersView.swift#L22). Am I missing something?

Reply:
Let's merge this in its current state, we can fix it after that in a separate PR.